### DISCUSSION TITLE
onFieldReact中设置异步数据源有时不生效

### DISCUSSION THREAD

**Discussion by DuanMo2017** [alibaba/formily#1817] [Q&A 1.x] [answered]
Select组件的demo中可以复现：https://codesandbox.io/s/cq8d6?file=/App.tsx
不是必现，比如搜这个词 "shitou" 时可以复现。是我打开方式有问题？🤔️
![image](https://user-images.githubusercontent.com/27721290/125922603-85cd2a79-fd03-4279-88ee-ae9446f34c92.png)

**Comment by janryWang** [comment, marked as answer]
Select传filterOption为false

**Reply by DuanMo2017** [reply]
OK ，解决了，感谢
